kong/api/routes/cluster.lua

```diff
@@ -33,7 +33,10 @@ return {
         return responses.send_HTTP_BAD_REQUEST("Missing node \"name\"")
       end
 
-      local _, err = singletons.serf:invoke_signal("force-leave", {self.params.name})
+      local inspect = require "inspect"
+      print(inspect(self.params.name))
```

Comment:
that seems like a left-over debug line, though not sure I'm not very familiar with the api side framework yet. @thibaultCha will know